arkouda_cws_hashing/server/lshMinMaxMsg.chpl

```diff
@@ -0,0 +1,79 @@
+module lshMinMaxMsg
+{
+
+  use lshMinMax;
+  use ServerConfig;
+  use MultiTypeSymEntry;
+  use ServerErrorStrings;
+  use MultiTypeSymbolTable;
+  use Message;
+  use Reflection;
+  use ServerErrors;
+  use Logging;
+
+
+  private config const logLevel = ServerConfig.logLevel;
+  const cwsLogger = new Logger(logLevel);
+
+
+  /* An implementation of a locality-sensitive hashing scheme for the MinMax (i.e. weighted 
+     Jaccard) kernel originally described in "Consistent Weighted Sampling" (Manasse, McSherry
+     Talwar). The specific implementation is an equivalent version by Sergie Ioffe described 
+     in "Improved Consistent Sampling, Weighted Minhash, and L1 Sketching" that improves the
+     runtime of the hashing setp from expected constant time to constant time. */
+
+
+  proc lshMinMaxMsg(cmd: string, payload: string, st: borrowed SymTab): MsgTuple throws {
+
+      param pn = Reflection.getRoutineName();
+
+      var (offsets, elts, weights, zbit, hashes) = payload.splitMsgToTuple(5);
+
+      const zBit: bool = zbit.toLower() == "true";
+      var numHashes = try! hashes: int(64);
```

Comment:
I think this will need to be changed slightly for the updated message passing code. Something like
```chapel
proc lshMinMaxMsg(cmd: string, payload: string, argSize: int, st: borrowed SymTab): MsgTuple throws {
    param pn = Reflection.getRoutineName();
    var msgArgs = parseMessageArgs(payload, argSize);
    const offsets = msgArgs.getValueOf("offsets")
    const elts = msgArgs.getValueOf("elts")
    const weights= msgArgs.getValueOf("weights")
    const zBit= msgArgs.get("zbit").getBoolValue();
    const numHashes= msgArgs.get("hashes").getIntValue();
```